src/components/MoneyReportHeader.js

```diff
@@ -51,39 +53,47 @@ const defaultProps = {
     },
 };
 
-function MoneyReportHeader(props) {
-    const moneyRequestReport = props.report;
+function MoneyReportHeader({session, personalDetails, policies, chatReport, report: moneyRequestReport, isSmallScreenWidth}) {
+    const {translate} = useLocalize();
+    const policy = policies[`${ONYXKEYS.COLLECTION.POLICY}${moneyRequestReport.policyID}`];
```

Comment:
I think that sounds like a good performance improvement to be honest

Reply:
Agreed, on it.

Reply:
@mountiny I've been trying to refactor components that have all policies in props instead of just one they need and I reached the component [ReportScreen](https://github.com/Expensify/App/blob/9a599aa718c48b9c7c68427cafd97810175fbf08/src/pages/home/ReportScreen.js#L419). Do you know why the report screen component of a single report should be receiving ALL policies? I looked into how policies are used in this component and we're always only concerned about the current report's policy, so why not just fetch that one?

Reply:
> Do you know why the report screen component of a single report should be receiving ALL policies? I looked into how policies are used in this component and we're always only concerned about the current report's policy, so why not just fetch that one?

To my knowledge I dont think you have to care about other policies there I agree